packages/artifactor/index.ts

```diff
@@ -7,8 +7,11 @@ const debug = require("debug")("artifactor");
 
 class Artifactor {
   destination: string;
-  constructor(destination: string) {
+  errorOnDuplicates: boolean;
```

Comment:
Let's just turn this behavior on in general so you can get rid of the `errorOnDuplicates` in this class.